lib/matplotlib/axes/_axes.py

```diff
@@ -4104,112 +4095,132 @@ def hexbin(self, x, y, C=None, gridsize=100, bins=None,
 
-        *edgecolors*: [ *None* | ``'none'`` | mpl color | color sequence ]
-           If ``'none'``, draws the edges in the same color as the fill color.
-           This is the default, as it avoids unsightly unpainted pixels
-           between the hexagons.
+        extent : scalar, optional, default is *None*
+            The limits of the bins. The default assigns the limits
+            based on *gridsize*, *x*, *y*, *xscale* and *yscale*.
 
-           If *None*, draws the outlines in the default color.
+            If *xscale* or *yscale* is set to 'log', the limits are
+            expected to be the exponent for a power of 10. E.g. for
+            x-limits of 1 and 50 in 'linear' scale and y-limits
+            of 10 and 1000 in 'log' scale, enter (1, 50, 1, 3).
 
-           If a matplotlib color arg or sequence of rgba tuples, draws the
-           outlines in the specified color.
+            Order of scalars is (left, right, bottom, top).
 
-        Here are the standard descriptions of all the
-        :class:`~matplotlib.collections.Collection` kwargs:
+        Other parameters
+        ----------------
+        cmap : object, optional, default is *None*
+            a :class:`matplotlib.colors.Colormap` instance. If *None*,
+            defaults to rc ``image.cmap``.
+
+        norm : object, optional, default is *None*
+            :class:`matplotlib.colors.Normalize` instance is used to
+            scale luminance data to 0,1.
+
+        vmin : scalar, optional, default is *None*
+            *vmin* is used in conjunction with *norm* to normalize
+            luminance data. If *None*, the min of the color array *C*
+            is used.  Note if you pass a norm instance, your settings
+            for *vmin* will be ignored.
+
+        vmax : scalar, optional, default is *None*
+            *vmax* is used in conjunction with *norm* to normalize
+            luminance data. If *None*, the max of the color array *C*
+            is used.  Note if you pass a norm instance, your settings
+            for *vmax* will be ignored.
+
+        alpha : scalar between 0 and 1, optional, default is *None*
+            the alpha value for the patches
+
+        linewidths : scalar, optional, default is *None*
+            If *None*, defaults to 1.0. Note that this is a tuple, and
+            if you set the linewidths argument you must set it as a
+            sequence of floats, as required by
+            :class:`~matplotlib.collections.RegularPolyCollection`.
+
+        edgecolors : {'none'}, mpl color, color sequence, optional,
+        default is 'none'
+            If 'none', draws the edges in the same color as the fill color.
+            This is the default, as it avoids unsightly unpainted pixels
+            between the hexagons.
+
+            If *None*, draws the outlines in the default color.
+
+            If a matplotlib color arg or sequence of rgba tuples, draws the
+            outlines in the specified color.
```

Comment:
Again, for hexbin, specifying anything other than a single color makes no sense: first, because one doesn't know at this point where the hexagon getting that edge color will be, and second, because even if one did, it would be nonsensical from a plotting standpoint.  So please omit everything having to do with the color sequence.